proto/viam/app/v1/app.proto

```diff
@@ -624,6 +627,19 @@ message DeleteRobotPartRequest {
   string part_id = 1;
 }
 
+message GetRobotAPIKeysRequest {
+  string robot_id = 1;
+}
+
+message ApiKey {
+  string key = 1;
+  string id = 2;
+}
```

Comment:
ah ok if we're adding an object, should we include the optional fields in the schema? e.g. `created_by`, `name` etc. or should we have a separate protobuf definition for those APIs?

Reply:
We can add created_by, name, and robot/location resource count as well. I had issues loading the figma design till now, but now I can see those fields. All of those except for name can be populated now, but we can just return "name" as an empty string till then. When are we planning to add the name field? as I don't see it as a field in the DB entry for IdentityAuth and I can't find the ticket